### PR TITLE
Replace most btn_solid with btn_outline and btn_primary + some misc. theme work

### PR DESCRIPTION
**Pull request by michaelkirk**
I'd recommend reviewing each commit individually - it starts with a grab bag of small fixes and one big commit at the end which converts most btn_solid into either an outline or primary button depending on it's usage.

Here's a tiny overview of the changes:

https://user-images.githubusercontent.com/217057/109344884-33507a00-7824-11eb-8170-40196598ff57.mp4

https://user-images.githubusercontent.com/217057/109344894-35b2d400-7824-11eb-8712-8cddc68393be.mp4

https://user-images.githubusercontent.com/217057/109344900-36e40100-7824-11eb-8894-b169f6cdcbd5.mp4

### CODE REVIEW

File: game/src/challenges/cutscene.rs [hunk 1]
```diff
@@ -263,7 +263,7 @@ impl FYI {
                 Widget::custom_col(vec![
                     contents,
                     ctx.style()
-                        .btn_solid_text("Okay")
```
Comment (michaelkirk) [inline game/src/challenges/cutscene.rs:266]: In most other places we say "OK", so converged to that.

File: game/src/pregame.rs [hunk 2]
```diff
@@ -360,14 +360,15 @@ impl Proposals {
                 if edits.proposal_link.is_some() {
                     current_tab.push(
                         ctx.style()
-                            .btn_solid_text("Read detailed write-up")
```
Comment (michaelkirk) [inline game/src/pregame.rs:363]: The link for this currently 404's. Maybe related to the org move?

https://dabreegster.github.io/abstreet/case_studies/west_seattle.html
https://dabreegster.github.io/abstreet/case_studies/lake_wash.html

Want me to file an issue?

Comment (michaelkirk) [reply game/src/pregame.rs:363]: I think in general we want to style hyperlinks as underlined text like this:

Screen Shot 2021-02-26 at 11 20 38 AM

File: game/src/sandbox/mod.rs [hunk 3]
```diff
@@ -444,7 +444,7 @@ impl AgentMeter {
                 .centered_vert(),
                 format!("{} trips captured", prettyprint_usize(n)).draw_text(ctx),
                 ctx.style()
-                    .btn_solid_text("Stop")
+                    .btn_solid_primary_text("Finish Capture")
```
Comment (michaelkirk) [inline game/src/sandbox/mod.rs:447]: Took some liberties with the copy... "Stop" is sort of ambiguous as to whether it means "complete" or "cancel"

**Review by dabreegster — APPROVED** [review]
LGTM! I'm assuming the older methods like `btn_solid_text` will get purged later?

**Comment by michaelkirk** [comment]
>  I'm assuming the older methods like btn_solid_text will get purged later?

They're still used in the tabs, but will likely go away once the tab redesign is finished. I'm not intending to complete that this sprint though.

**Pull request merged**